### DISCUSSION TITLE
Testing with spaces in labels

### DISCUSSION THREAD

**Discussion by Foxite** [hneemann/Digital#1206] [Q&A] [answered]
I'm trying to test a processor where I have labeled various elements like "program counter" "A register" etc. When using test vectors, I wasn't able to find out how to use an element with a label that contains a space.

I would have used underscores but this causes the part after the underscore to be formatted as a subscript, which is not always what I want. Is there a way to prevent this for individual elements?

Of course I could just rename them with camelCase. But is there a way to use test vectors when your labels have spaces in them?

**Comment by hneemann** [comment, marked as answer]
> But is there a way to use test vectors when your labels have spaces in them?

Unfortunately no.
